Old helm version in docker image doesn't support lookup function

**Is this a request for help?**: No

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

**Version of Helm and Kubernetes**:

```
$ helm version
version.BuildInfo{Version:"v3.2.4", GitCommit:"0ad800ef43d3b826f31a5ad8dfbb4fe05d143688", GitTreeState:"clean", GoVersion:"go1.13.12"}

$ kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.1", GitCommit:"7879fc12a63337efff607952a323df90cdc7a335", GitTreeState:"clean", BuildDate:"2020-04-08T17:38:50Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-30T20:19:45Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}

$ ct version
Version:	 v3.0.0
Git commit:	 50db473a1e68c605b18d82f019d83ea401542213
Date:		 2020-07-24T20:35:05Z
License:	 Apache 2.0
```

**What happened**:
I ran `ct lint` locally. My chart passed.

I created a PR which ran the chart-testing github action (https://github.com/helm/chart-testing-action). My chart failed with:
```
[ERROR] templates/: parse error at (lagoon-core/templates/keycloak.secret.yaml:7): function "lookup" not defined
```

**What you expected to happen**:
I expected my chart to pass the `ct lint` check.

**How to reproduce it** (as minimally and precisely as possible):
Run the chart-testing github action against a chart which uses the `lookup` function.

**Anything else we need to know**:
I've figured out that the reason that this is failing is that the version of `helm` in the image is too old. Will submit a PR to fix.

--- a/Dockerfile
+++ b/Dockerfile
@@ -28,7 +28,7 @@ RUN curl -LO "https://storage.googleapis.com/kubernetes-release/release/$kubectl
     mv kubectl /usr/local/bin/
 
 # Install Helm
-ARG helm_version=v3.1.2
+ARG helm_version=v3.2.4
 LABEL helm_version=$helm_version
 RUN curl -LO "https://get.helm.sh/helm-$helm_version-linux-amd64.tar.gz" && \
     mkdir -p "/usr/local/helm-$helm_version" && \

